assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -65,8 +68,16 @@ function DashboardSummaryWidget() {
 			metrics,
 		};
 
+		// Get the first day of the month an ISO 8601 date string.
+		const startOfMonth = new Date(
+			new Date( referenceDate ).getFullYear(),
+			new Date( referenceDate ).getMonth(),
+			1
+		).toLocaleDateString( 'sv-SE' ); // Sweden uses ISO 8601 as their official date format.
```

Comment:
So, that's neat, but my concern about this is… what happens if Sweden changes their date format? 😆 

I know it's unlikely, but this feels like using the fact that Swedish in Sweden uses that format as a hack rather than outputting exactly what we meant to output clearly.

Is the main reason for doing this to avoid using `Intl.DateTimeFormat`? What's the reason behind it?

This is a case of "shorter code" but less clear code to me, so I'd rather we didn't rely on a specific locale's implementation for our internal date formats 😄

Reply:
Hey @tofumatt, the issue with the `Intl.DateTimeFormat` implementation is that, if set to `'en'`, it will return dates as: `1/7/2021`, whereas the API is looking for an ISO 8601 data which is delineated by dashes not slashes and is in reverse order: `2021-01-07`. 

I get your point about this solution though. Another more reliable solution might be to get a substring of:

```
new Date().toISOString()
```

Which outputs:

```
2021-01-07T12:13:46.567Z
```

Reply:
Ah, yeah, I think that's a bit better and more clear regarding intent. Happy to go with that 👍🏻

Reply:
Update and back over to you.